### PR TITLE
docs: remove example notebooks from this repository.

### LINKED ISSUE #533: Remove example notebooks
These are now stored in a separate documentation repo.

### PR DESCRIPTION
**Pull request by CalMacCQ**
# Description

I'm removing the `examples` directory as we now have the content in a nicer form on the website. We should direct users there instead.

See
1. [Knowledge articles](https://docs.quantinuum.com/h-series/trainings/knowledge_articles/ka_index.html)
2. [pytket-quantinuum section](https://docs.quantinuum.com/h-series/trainings/getting_started/pytket_quantinuum/pytket_quantinuum.html) 

To my knowledge, there is no example content that is in the github repo that we don't have on the docs.quantinuum.com website. Do you agree @irfankhan10?

I've also updated some links, making sure to link to the quantinuum docs website rather than the github repo.

Maybe we should also come up with a way to still run these example notebooks in the other repo with the latest pytket-quantinuum version to detect issues. We could set up a cronjob or something to do this. Anyway that's for another PR.

# Related issues

closes #533 

# Checklist

- [x] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

File: README.md
Comment (CalMacCQ) [inline README.md:29]: Maybe I should link to specific subsections from [this page](https://docs.quantinuum.com/h-series/trainings/getting_started/pytket_quantinuum/pytket_quantinuum.html) as well as the knowledge articles? However that would lead to a lot of links in the README.

**Comment by irfankhan10** [comment]
@cqc-alec and @cqc-melf , I would like to unit tests for the QuantinuumCalendar method, before merging this PR. Should those tests be part of this PR or seperate?

**Review by cqc-alec — APPROVED** [review]
Just a small comment but LGTM!

**Comment by cqc-alec** [comment]
> @cqc-alec and @cqc-melf , I would like to unit tests for the QuantinuumCalendar method, before merging this PR. Should those tests be part of this PR or seperate?

I think that should be a separate PR.

File: README.md
Comment (CalMacCQ) [reply README.md:31]: done [fc8b274](https://github.com/CQCL/pytket-quantinuum/pull/543/commits/fc8b2748f5ccbce59fc17f8d66b0b50b410d4482)

File: README.md
Comment (irfankhan10) [reply README.md:29]: Will defer to @cqc-alec, but I think easier for only one link to exist from H-Series to the TKET page.

Comment (CalMacCQ) [reply README.md:29]: I'll leave it as is for now

**Review by irfankhan10 — APPROVED** [review]

**Pull request merged**
